packages/swapper/src/thorchain-utils/memo/assertAndProcessMemo.test.ts

```diff
@@ -104,7 +104,7 @@ describe('assertAndProcessMemo', () => {
       expect(() => assertAndProcessMemo(memo)).toThrow()
     })
 
-    it('should throw on invalid destination address', () => {
+    it('should throw on missing destination address', () => {
```

Comment:
This was incorrect, we don't do address validation, we're really testing for missing address here